File: LABORATORIOS/LABORATORIO2/Ejercicio7/Python/Fibonacci.py
def calcular_producto_fibonacci(n):
    if n <= 0:
        return 0

    producto = 1
    a, b = 0, 1

    for _ in range(n):
        producto *= a
        a, b = b, a + b

    return producto

n_terminos = 8
producto_fibonacci = calcular_producto_fibonacci(n_terminos)
print(producto_fibonacci)

File: LABORATORIOS/LABORATORIO2/Ejercicio7/Java/Fibonacci.java
public class Fibonacci {
    public static int calcularProductoFibonacci(int n) {
        if (n <= 0) {
            return 0;
        }

        int producto = 1;
        int a = 0;
        int b = 1;

        for (int i = 0; i < n; i++) {
            producto *= a;
            int temp = a;
            a = b;
            b = temp + b;
        }

        return producto;
    }

    public static void main(String[] args) {
        int nTerminos = 8;
        int productoFibonacci = calcularProductoFibonacci(nTerminos);
        System.out.println(productoFibonacci);
    }
}